components/style/properties/longhand/font.mako.rs

```diff
@@ -1813,22 +1820,31 @@ ${helpers.single_keyword_system("font-variant-position",
                     return Err(())
                 }
 
+                let b_tag = tag.deref().as_bytes();
```

Comment:
It had a type error when I attempted to compile without deref. Stated that COW did not have as_bytes defined on it. Adding an explicit deref made the error go away.